schemas/common.json

```diff
@@ -284,29 +284,36 @@
             "sectionEnd": "vCenter infrastructure",
             "propertyOrder": 340
         },
+        "vcenter_vmfolder": {
```

Comment:
This new item is part of the vCenter infrastructure section in the schema. Line 284, that defines the end of the section, needs to me moved to this entry so that all vCenter params are properly sectioned.